<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>271</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Решение квадратного уравнения</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="a_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>a=</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>b=</string>
    </property>
   </widget>
   <widget class="QLabel" name="c_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>68</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>c=</string>
    </property>
   </widget>
   <widget class="QLabel" name="descrption">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Корни квадраного уравнения вида ax^2+bx+c=0:</string>
    </property>
   </widget>
   <widget class="QLabel" name="answer">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>381</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>отсутствуют</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="x1_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x1=</string>
    </property>
   </widget>
   <widget class="QLabel" name="x2_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>31</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>x2=</string>
    </property>
   </widget>
   <widget class="QLabel" name="x1_num">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>num1</string>
    </property>
   </widget>
   <widget class="QLabel" name="x2_num">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>num2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Act_Button">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="a_SpinBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="b_SpinBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="c_SpinBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
